src/pages/manualLoggingSteps/manualLoggingSteps.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import {
+    IonButton,
+    IonCol,
+    IonContent,
+    IonHeader,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonNote,
+    IonRouterLink,
+    IonTitle,
+    IonToolbar,
+    IonGrid,
+    IonRow,
+} from '@ionic/react';
+//import ExploreContainer from '../../components/ExploreContainer';
+import './manualLoggingSteps.css';
+
+const ManualSteps: React.FC = () => {
+    const [username, setUsername] = useState();
+    const [password, setPassword] = useState();
+    const [isTouched, setIsTouched] = useState(false);
+    const [isValid, setIsValid] = useState<boolean>();
+    var stepsToLog: number;
+    var date: Date;
+    // var pastRecordDates: Array: Date;
```

Comment:
Do we still need this commented out code?

Reply:
Nope, Will be removed